docs/quickstart.md

```diff
@@ -0,0 +1,47 @@
+# Quickstart
+The following instructions build an index from our scientists example Knowledge
+Base
+
+## Download and build QLever using `docker`
+
+    git clone --recursive https://github.com/ad-freiburg/QLever.git qlever
+    cd qlever
+    docker build -t qlever .
+
+## Download the example Knowledge Base
+
+    wget http://filicudi.informatik.uni-freiburg.de/bjoern-data/scientist-collection.zip
+    unzip -j scientist-collection.zip -d scientists
+
+## Build a QLever Index
+
+    docker run -it --rm \
+        -v "$(pwd)/scientists:/input" \
+        -v "$(pwd)/index:/index" --entrypoint "bash" qlever
+    qlever@xyz:/app$ IndexBuilderMain -a -l -i /index/scientists \
+        -n /input/scientists.nt \
+        -w /input/scientists.wordsfile.tsv \
+        -d /input/scientists.docsfile.tsv
+    qlever@xyz:/app$ exit
+
+## Run QLever
+
+    docker run -it -p 7001:7001 \
+        -v "$(pwd)/index:/index" \
+        -e INDEX_PREFIX=scientists \
+        --name qlever \
+        qlever
+
+Then point your browser to [http://localhost:7001/](http://localhost:7001/) and
+enter the query.
+
+For example all female scientists occuring in the text with "algo*" can be
+obtained with the following
```

Comment:
`in the text` -> `in the text corpus` as no text was mentioned right before this sentence. `with "algo*"` -> `with the regex "algo*"`
`with the following` -> `with the folllowing query`